cEP-0018.md

```diff
@@ -0,0 +1,162 @@
+# Integration of ANTLR into coala core
+
+| Metadata     |                                         |
+| ------------ |-----------------------------------------|
+| cEP          | 0018                                    |
```

Comment:
change that to 18 no need for the two 0's

Reply:
+1

Reply:
comments on https://github.com/coala/cEPs/issues/120 pls @palash25 & @Udayan12167